text/0000-router-link-and-helpers.md

```diff
@@ -0,0 +1,161 @@
+- Start Date: 2018-06-15
+- RFC PR:
+- Ember Issue:
+
+# Add a Link component and routing helpers
+
+## Summary
+
+The purpose of this is to add a single component, `Link`, that wraps up some routing helpers for default usage of linking / routing.
+
+## Motivation
+
+Currently, on ember#canary, the Angle Bracket Invocation feature doesn't support positional arguments. For clarity, and with some inspiration from react-router, It may be beneficial for Ember to include a `Link` component, similar to the existing `LinkTo` (`link-to`), but with only named arguments. This will help with clarity of intent, and also allow for easier transitioning from other communities who already have named-argument links.
```

Comment:
Positional arguments are not planned for angle bracket invocations, I suggest updating this sentence to remove any "but in the future it might" implicit suggestion.